src/annotator/anchoring/test/pdf-test.js

```diff
@@ -85,7 +85,6 @@ describe('annotator/anchoring/pdf', () => {
   });
 
   afterEach(() => {
-    pdfAnchoring.$imports.$restore();
```

Comment:
I was not sure about this change. I don't see any imports being mocked, so I thought it is not needed.

Reply:
There is a call to `$imports.$mock` in one specific test currently. While it would be possible to use `try ... finally` in that test, I'd consider it idiomatic to have a top-level `$imports.$restore` call if any test makes use of it.